### ISSUE TITLE
onwheels / accessibility and quality for wheelchair missing

### ISSUE THREAD

**Issue by JLZIMMERMANN** [pietervdvn/MapComplete#1195]
**FACT** : onwheels map displays a lot of elements as doors, shops, kerbs, parking space

**IDEA** : it would be welcome to see more the properties of these elements for people suing wheelchair
let the map show tag wheelchair=* values waited at least yes,designed=GREEN no=RED bad,half,limited=YELLOW

**ALREADY DONE** : in wheelmap.org
![image](https://user-images.githubusercontent.com/4929752/209535019-bc64a562-bddd-4b60-8b7b-42a207255ab3.png)

**Comment by thibaultmol** [comment]
From my convo with pieter and robin (from onwheels):

The issue is much more complex. There would have to be a whole system where users can specify
- their exact type of disabilities
- what kind of wheel chair they have
- what slope angles they're comfortable with
- how wide their wheelchair is
Etc...

So just implemented a simple color system like wheelmap is.... Useless except for a very specific type of user

**Comment by JLZIMMERMANN** [comment]
I understand but quality start with this : try and accurate.
OSM let us get this and it would be welcome to do maybe like putting a colored dot
![image](https://user-images.githubusercontent.com/4929752/209536714-9db4ef9f-74ad-4162-b464-83818e3c9e78.png)

**Comment by pietervdvn** [comment]
OnWHeels does exactly this: add physical measurements (such as width, kerb height, automation) onto the relevant features such as doors, kerbs, elevators.

It is up to the data user (the wheelchair user, or the traveller with their bike, or someone pushing a stroller) to decide if a door/shop/elevator is suited for them.

It seems like all ideas are already implemented, so I'm closing this issue.
